modules/core/private_channel/member_controller.py

```diff
@@ -82,7 +88,13 @@ def autoinvite_cmd(self, request, pref):
             pref = self.getresp("module/private_channel", "on" if pref == "on" else "off")
             return self.getresp("module/private_channel", "autoinvite_changed", {"changedto": pref})
         else:
-            return self.getresp("module/private_channel", "not_an_member")
+            if self.access_service.check_access(request.sender.char_id, self.org_member_controller.ORG_ACCESS_LEVEL):
```

Comment:
it would be better not to hard-code the access level that this will work for, and instead just relay on whatever access level is set for this command by the bot owner.  that way if they want to open it up to the public, they can just by changing the access level.

Reply:
and in that case, it may not be necessary to tell them they have been added as a member, since that is somewhat of an implementation detail and most people won't know what that means, or care.  but i'll leave it up to you to decide if you think there is value with the additional message.